### PR TITLE
Feature/market refinement

### PR DESCRIPTION
**Pull request by zachdt**
This is an incomplete PR, there are a few things I wanted to finish before V1, but due to personal circumstances I am opening this preemptively. 

TODO:
- Slippage Settings are working, but the useSlippage hook isn't used in the lib yet
- TransactionCard is functional, but completely un-styled
- LP Provision/Withdraw views are non-functional, but are styled
- Gas Limit bug for purchase/selling option tokens is still prevalent
- Position Card is incomplete, I attempted to create a new useBalance hook based on [hypertext](https://github.com/NoahZinsmeister/hypertext/blob/18af671adf6fa517b84310a6d21f3780e624c864/hooks.ts), but there are limitations to using useSWR inside another hook.  The useContract hook is working, so balances can be grabbed from option contracts, but the hook using useContract is non-functional.
- Positions inside the position card should be clickable, and count as adding an OrderItem to the order context.

### CODE REVIEW

**Comment by zachdt** [comment]
Just remembered, there are a couple global Next.js styling bugs I can squash in this PR fairly easily (double scrollbar, weird global margin).

**Comment by zachdt** [comment]
Also need a footer!  I can work on this, my power hasn't gone out quite yet.

File: app/src/components/BetaBanner/BetaBanner.tsx [hunk 1]
```diff
@@ -19,12 +19,10 @@ const BetaBanner: React.FC<BetaBannerProps> = ({ isOpen }) => {
   if (!open) return null
   return (
     <StyledContainer>
-      <CloseButton onClick={handleClose} variant="transparent" size="sm">
-        <CloseIcon fontSize="small" />
-      </CloseButton>
       <StyledLink href="https://primitive.finance/beta">
         <StyledTitle>
-          Earn NFT Rewards
+          Make a Trade, Earn NFT Rewards
```
Comment (Alexangelj) [inline app/src/components/BetaBanner/BetaBanner.tsx:24]: Note for me: Earn Rewards

File: app/src/components/BetaBanner/BetaBanner.tsx [hunk 2]
```diff
@@ -51,15 +49,14 @@ const StyledTitle = styled.h3`
...
+  padding: 0em;
   flex-direction: row;
-  border-radius: 1em;
+  border-radius: 0.2em;
   justify-content: center;
-  height: 5em;
-  position: fixed;
-  width: 10em;
+  height: 3em;
+  width: 100%;
```
Comment (Alexangelj) [inline app/src/components/BetaBanner/BetaBanner.tsx:59]: Note for me: 
- alphabetize
- conditional styling

Comment (Alexangelj) [reply app/src/components/BetaBanner/BetaBanner.tsx:59]: oops doesnt need review

File: app/src/components/Settings/Settings.tsx [hunk 3]
```diff
@@ -20,6 +21,9 @@ export const Settings = () => {
   const nodeRef = useClickAway(() => {
     setOpen(false)
   })
+  const handleSlip = (s: string) => {
+    setSlippage(s)
+  }
```
Comment (Alexangelj) [inline app/src/components/Settings/Settings.tsx:26]: nice

File: app/src/components/Settings/Settings.tsx [hunk 4]
```diff
@@ -82,8 +125,8 @@ const StyledModal = styled.div`
   color: ${(props) => props.theme.color.white};
   padding: ${(props) => props.theme.spacing[3]}px;
   position: fixed;
-  right: 15%;
-  top: ${(props) => props.theme.barHeight}px;
+  right: 10%;
+  top: ${(props) => props.theme.barHeight + 10}px;
```
Comment (Alexangelj) [inline app/src/components/Settings/Settings.tsx:129]: should probably make this more programmatic

File: app/src/components/TopBar/TopBar.tsx [hunk 5]
```diff
@@ -25,7 +25,7 @@ const TopBar: React.FC = () => {
           justifyContent="flex-start"
           display="flex"
           flexDirection="row"
-          height={65}
+          height={75}
         >
```
Comment (Alexangelj) [inline app/src/components/TopBar/TopBar.tsx:29]: why 75? I like 72

**Review by Alexangelj — COMMENTED** [review]
LGTM

**Review by Alexangelj — APPROVED** [review]

File: app/src/components/Footer/Footer.tsx [hunk 6]
```diff
@@ -1,30 +1,58 @@
...
+      </StyledBlockNumber>
+    </StyledFooter>
+  )
+}
 
+const breathe = keyframes`
+  0% { opacity: .5;}
+  50% {opacity: 1; }
+  100% { opacity: .5;}
+;`
```
Comment (Alexangelj) [inline app/src/components/Footer/Footer.tsx:26]: yessss

File: app/src/components/Footer/Footer.tsx [hunk 7]
```diff
@@ -1,30 +1,58 @@
...
   display: flex;
   justify-content: center;
   height: ${(props) => props.theme.topBarSize}px;
   max-width: ${(props) => props.theme.siteWidth}px;
-  width: 100%;
-  position: fixed;
+
   margin-right: ${(props) => props.theme.spacing[4]}px;
   padding: ${(props) => props.theme.spacing[4]}px;
 `
```
Comment (Alexangelj) [inline app/src/components/Footer/Footer.tsx:58]: Note for me:
- alphabetize
- conditional styling

File: app/src/components/TopBar/TopBar.tsx (hunk 5 above)
Comment (zachdt) [reply app/src/components/TopBar/TopBar.tsx:29]: 75 seems to be the most centered, but I could be wrong.  as long as its >70 it looks fine

**Review by Alexangelj — APPROVED** [review]

File: app/src/components/Footer/Footer.tsx (hunk 7 above)
Comment (zachdt) [reply app/src/components/Footer/Footer.tsx:58]: Should probably fix the footer to the bottom, shouldn't have to scroll down to see anything.

**Pull request merged**